### PR TITLE
DOC-2411: TinyMCE 7.1.1 Release Notes and Changelog.

### PR DESCRIPTION
**Pull request by FarzadHayat**
Ticket: DOC-2411

Site: [Staging branch](http://docs-feature-711-doc-2411.staging.tiny.cloud/docs/tinymce/latest/7.1.1-release-notes)

Changes:
* DOC-2411: 7.1.1 Release Notes template outline

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`
- [x] `modules/ROOT/nav.adoc` has been updated `(if applicable)`
- [x] Files has been included where required `(if applicable)`
- [x] Files removed have been deleted, not just excluded from the build `(if applicable)`
- [x] Files added for `New product features`, and included a `release note` entry.
- [x] Fix merge conflict in `release-notes.adoc` after TinyMCE 7.1 is released.
- [x] Fix merge conflict in `nav.adoc` after TinyMCE 7.1 is released.
- [x] Update `changelog.adoc` to include 7.1.1 entries.
- [x] Re-update `changelog.adoc` after missing tickets have been backported.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/7.1.1-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+The following premium plugin updates were released alongside {productname} {release-version}.
+
+=== PowerPaste
+
+The {productname} {release-version} release includes an accompanying release of the **PowerPaste** premium plugin.
+
+**PowerPaste** includes the following fix.
+
+==== A list with the start attribute, separated by `p` tags, resulted in incorrect numbering of lists.
```
Comment (spocke) [inline modules/ROOT/pages/7.1.1-release-notes.adoc:33]: We decided to move this to 7.2

File: modules/ROOT/pages/changelog.adoc [hunk 2]
```diff
@@ -4,66 +4,80 @@
...
+// #TINY-10878
+* Iframes in dialogs were not rendering rounded borders correctly.
+// #TINY-10901
+* Autocompleter possible values are no longer capped at a length of 10.
+// #TINY-10942
+
 == 7.1 - 2024-05-08
 
-### Added
+=== Added
```
Comment (spocke) [inline modules/ROOT/pages/changelog.adoc:22]: I assume that this was a mistake that we made on the previous 7.1 release?

**Review by spocke — APPROVED** [review]

File: modules/ROOT/pages/7.1.1-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+* xref:improvements[Improvements]
+* xref:additions[Additions]
+* xref:changes[Changes]
+* xref:bug-fixes[Bug fixes]
+* xref:security-fixes[Security fixes]
+* xref:deprecated[Deprecated]
+* xref:known-issues[Known issues]
+
+
+[[new-premium-plugin<s>]]
```
[the remark this answers is not in the document]
Comment (kemister85) [reply modules/ROOT/pages/7.1.1-release-notes.adoc]: All boilerplate content is removed before we merge back into the target release/7.1.1 branch.

File: modules/ROOT/pages/7.1.1-release-notes.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/pages/7.1.1-release-notes.adoc:33]: When was the target version changed? Happy to remove if we need it to.

File: modules/ROOT/pages/changelog.adoc (hunk 2 above)
Comment (kemister85) [reply modules/ROOT/pages/changelog.adoc:22]: yeah, just a `asciidoc` vs `md` syntax error.

File: modules/ROOT/pages/7.1.1-release-notes.adoc (hunk 1 above)
Comment (ShiridiGandham) [reply modules/ROOT/pages/7.1.1-release-notes.adoc:33]: @kemister85 Please remove this entry. It is now scheduled to go out with 7.2.

File: modules/ROOT/pages/7.1.1-release-notes.adoc [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+[source]
+----
+7. Point 1 
+7. Point 2
+7. Point 3
+----
+
+{productname} {release-version} addresses this issue. Now, all lists correctly render with the proper numbering.
+
+For information on the **PowerPaste** plugin, see: xref:introduction-to-powerpaste.adoc[PowerPaste].
```
Comment (kemister85) [inline modules/ROOT/pages/7.1.1-release-notes.adoc:49]: Removed as this was moved to `7.2`.
```suggestion
```

File: modules/ROOT/pages/7.1.1-release-notes.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (FarzadHayat) [reply modules/ROOT/pages/7.1.1-release-notes.adoc]: Done ✅

[the remark this answers is not in the document]
Comment (kemister85) [reply modules/ROOT/pages/7.1.1-release-notes.adoc]: Also re-arranged the entries to be inline with the changelog.adoc @FarzadHayat

**Review by kemister85 — APPROVED** [review]

**Comment by FarzadHayat** [comment]
@ShiridiGandham need your approval before merging

**Review by ShiridiGandham — APPROVED** [review]
LGTM

**Pull request merged**